File: 242.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] mapper = new int[26];
        for(char ch: s.toCharArray()) {
            int index = ch - 'a';
            mapper[index] += 1;
        }

        for(char ch: t.toCharArray()) {
            int index = ch - 'a';
            mapper[index] -= 1;
        }

        for(int i=0;i<26;i++) {
            if(mapper[i]!=0){
                return false;
            }
        }
        return true;
    }
}

File: 242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):
            return False

        mapper = [0]*26
        for c1,c2 in zip(s, t):
            mapper[ord(c1)-ord('a')] +=1
            mapper[ord(c2)-ord('a')] -= 1

        # Check if every value is 0 or not
        for num in mapper:
            if num != 0:
                return False
        return True